ecc/bn254/fr/poseidon/poseidon.go

```diff
@@ -1,11 +1,18 @@
 package poseidon
 
 import (
+	"errors"
 	"github.com/consensys/gnark-crypto/ecc/bn254/fr/poseidon/constants"
+	"hash"
+	"math/big"
 
 	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
 )
 
+const (
+	BlockSize = fr.Bytes // BlockSize size that mimc consumes
```

Comment:
BlockSize = fr.Bytes // BlockSize size that Poseidon consumes

Reply:
Fixed